apps/graphql/src/apps/booking/queries/__tests__/queries.test.js

```diff
@@ -7,6 +7,6 @@ import Itineraries from '../../../itinerary/__datasets__/Itineraries.json';
 
 fetch.mockResponses([JSON.stringify(Itineraries)]);
 
-generateTestsFromFixtures(`${__dirname}/__fixtures__`, input =>
+generateTestsFromFixtures(`${__dirname}/__fixtures__/queries`, input =>
```

Comment:
Please don't do this, we don't have to copy paste this file every time we add a new test. The idea is that it reads all fixtures and creates snapshots.

Reply:
Done!, the separation is done in `mutations`, `queries`.